etc/make/local.mk

```diff
@@ -126,12 +126,12 @@ basiclint: gofmt govet golint staticcheck errcheck # run gofmt govet golint stat
 lint: basiclint generatenodiff nogogenerate verifyversion verifycodecovignores ## run all linters
 
 .PHONY: test
-test: $(THRIFTRW) __eval_packages ## run all tests
-	PATH=$(BIN):$$PATH go test -race $(PACKAGES)
+test: $(THRIFTRW) __eval_chunked_packages ## run chunked tests
+	PATH=$(BIN):$$PATH go test -race $(CHUNKED_PACKAGES)
 
 .PHONY: cover
-cover: $(THRIFTRW) $(GOCOVMERGE) $(PARALLEL_EXEC) $(COVER) __eval_packages ## run all tests and output code coverage
-	PATH=$(BIN):$$PATH ./etc/bin/cover.sh $(PACKAGES)
+cover: $(THRIFTRW) $(GOCOVMERGE) $(PARALLEL_EXEC) $(COVER) __eval_chunked_packages ## run chunked tests and output code coverage
+	PATH=$(BIN):$$PATH ./etc/bin/cover.sh $(CHUNKED_PACKAGES)
```

Comment:
Won't this basically stomp on coverage from different shards until the last one is just the tests from that shard?

Reply:
From the codecov docs: https://docs.codecov.io/v4.3.6/docs/merging-reports
> Codecov does not override report data for multiple uploads. We always merge the data. Simply upload all three reports at once, or separately.

So this should just merge the parallel codecov runs. I'll do some more verification on my branch, but this should just work.